move-vm-backend/build.rs

```diff
@@ -0,0 +1,40 @@
+use std::process::Command;
+
+fn main() {
+    // Tell Cargo that if the given file changes, to rerun this build script.
+    //println!("cargo:rerun-if-changed=../deposit/sources/deposit.move");
+    // SMOVE build our deposit code
+    Command::new("move")
+        .args(["build", "-p", "../deposit"])
+        .output()
+        .expect("failed to execute process");
+    // copy new binary module so it can be properly included
+    Command::new("cp")
+        .args([
+            "../deposit/build/deposit/bytecode_modules/deposit.mv",
+            "../contracts",
+        ])
+        .output()
+        .expect("failed to copy new deposit.mv");
+    // SMOVE build our deposit script
+    Command::new("move")
+        .args(["build", "-p", "../deposit/executor"])
+        .output()
+        .expect("failed to execute process");
+    println!("script copy");
```

Comment:
Uhuh, not sure if this is the prettiest approach. Could you check the alternative approach with the [move-stdlib](https://github.com/eigerco/substrate-move/pull/39)?

I wouldn't mind even if we move the Deposit module to the stdlib module directly (it's published under the same address, after all), and then we can get full access just by accessing the stdlib - then we wouldn't need any extra handling just for the Deposit module.